source/_integrations/wilight.markdown

```diff
@@ -0,0 +1,124 @@
+---
+title: WiLight
+description: Instructions on how to integrate WiLight devices into Home Assistant.
+logo: wilight.png
```

Comment:
Please add a logo to the [brand repository](https://github.com/home-assistant/brands)

Reply:
Done!
Please, can you check if it is OK?
Thank you!

Reply:
PR in brand repository #1553

Reply:
It needs to be removed from this PR...